common/src/main/resources/db/changelog/v001/ab2d-599-rename-consent.sql

```diff
@@ -5,11 +5,9 @@
 --changeset spathiyil:ab2d-599-rename-consent-table-to-opt_out failOnError:true
 ALTER TABLE consent RENAME TO opt_out;
 ALTER TABLE opt_out RENAME CONSTRAINT "pk_consent" TO "pk_opt_out";
-ALTER INDEX "ix_consent_hicn" RENAME TO "ix_opt_out_hicn";
 
 --rollback  ALTER TABLE opt_out RENAME TO consent;
```

Comment:
@lscarreon-sm don't touch these liquibase files. They've already been applied to the database. If there is something we need to remove from the database then create a sql script that does that and it will be applied on the next deployment.